asv_bench/benchmarks/groupby.py

```diff
@@ -132,3 +141,34 @@ def setup(self, *args, **kwargs):
         super().setup(**kwargs)
         self.ds1d = self.ds1d.chunk({"time": 50})
         self.ds2d = self.ds2d.chunk({"time": 50, "z": 4})
+
+
+class ResampleCFTime:
```

Comment:
```suggestion
class ResampleCFTime(Resample):
```
This class has currently no tests, is this what was intended?